components/taupage.rst

```diff
@@ -688,3 +688,25 @@ You can build your own Taupage AMI using the code from the repository on GitHub
 In the repository you will find a configuration (config-stups-example.sh) file which you'll have to adjust to your needs.
 
 See :ref:`taupage-ami-creation` for details.
+
+
+Using Nvidia gpus with nvidia-docker
++++++++++++++++++++++
+If gpu device files are found on the host the usage of nvidia-docker is triggered automatically. This has the benefit that drivers are automatically mounted into the docker container (there is no need to install identical drivers in the container).
```

Comment:
Oh no, I just started reading the PR and Hacker News thread... Docker (the company) is unbelievable... (not in a positive sense)